.github/workflows/ci-nightly.yml

```
@@ -28,19 +28,23 @@ jobs:
       run: |
         git config --global user.name "${{ env.GITHUB_SERVICE_USER }}"
         git config --global user.email "${{ env.GITHUB_SERVICE_EMAIL }}"
-    
+   
     - name: Install package dependencies / prepare workspaces
       run: yarn install --frozen-lockfile
+    
+    - name: check for change
```

This might be best as "check change" as this step is ensuring there is a change file - if not, we'd exit right away.

```suggestion
    - name: check change
```